extensions/bitwarden/src/create-login.tsx

```diff
@@ -0,0 +1,244 @@
+      const generatedPassword = await bitwarden.generatePassword(options);
+      updateField("password")(generatedPassword);
+      await Clipboard.copy(generatedPassword);
+      toast.title = "Password generated and copied";
+      toast.style = Toast.Style.Success;
+    } catch (error) {
+      toast.title = "Failed to generate password";
+      toast.style = Toast.Style.Failure;
+    }
+  };
+
+  return (
+    <Form
+      isLoading={isLoading}
+      actions={
+        <ActionPanel>
+          <Action.SubmitForm title="Create Login" onSubmit={handleSubmit(onSubmit)} icon={Icon.NewDocument} />
+          <Action
+            icon={showPassword ? Icon.EyeDisabled : Icon.Eye}
+            title={showPassword ? "Hide Password" : "Show Password"}
+            onAction={togglePasswordVisibility}
+            shortcut={{ modifiers: ["cmd"], key: "e" }}
+          />
+          <Action
+            icon={Icon.Key}
+            title="Generate Password"
+            onAction={generatePassword}
+            shortcut={{ modifiers: ["cmd"], key: "g" }}
+          />
+          <DebuggingBugReportingActionSection />
+        </ActionPanel>
+      }
+    >
+      <Form.TextField
+        id="name"
+        title="Name"
+        placeholder="eg: GitHub, Gmail"
+        value={formState.name}
+        onChange={updateField("name")}
+        onBlur={markFieldAsTouched("name")}
+        storeValue={false}
+        ref={nameFieldRef}
+        error={getFieldError("name")}
+      />
+      <Form.Dropdown
+        id="folder"
+        title="Folder"
+        placeholder="Select folder (optional)"
+        value={formState.folderId ?? FOLDER_OPTIONS.NO_FOLDER}
+        onChange={(value) => updateField("folderId")(value === FOLDER_OPTIONS.NO_FOLDER ? null : value)}
+        storeValue={false}
+      >
+        {folders.map((folder) => (
+          <Form.Dropdown.Item
+            key={folder.id}
+            value={folder.id ?? FOLDER_OPTIONS.NO_FOLDER}
+            title={folder.name}
+            icon={Icon.Folder}
+          />
+        ))}
+      </Form.Dropdown>
+      <Form.TextField
+        id="username"
+        title="Username"
+        placeholder="eg: john.doe@example.com"
+        value={formState.username}
+        onChange={updateField("username")}
+        onBlur={markFieldAsTouched("username")}
+        storeValue={false}
+        error={getFieldError("username")}
+      />
+      <PasswordField
+        id={showPassword ? "createLoginPlainPassword" : "createLoginPassword"}
+        title="Password"
+        placeholder="Enter password"
+        value={formState.password}
+        onChange={updateField("password")}
+        onBlur={markFieldAsTouched("password")}
+        ref={passwordFieldRef}
+        storeValue={false}
+        error={getFieldError("password")}
+      />
+      <Form.Description
+        title=""
+        text={`Press ⌘E to ${showPassword ? "hide" : "show"} password\nPress ⌘G to generate password`}
+      />
+    </Form>
```

Comment:
I feel like the `Website (URI)` field is important to have in this form, could you add it?